pytorch_lightning/accelerators/accelerator.py

```diff
@@ -338,21 +338,42 @@ def lightning_module_state_dict(self) -> Dict[str, Union[Any, Tensor]]:
         return self.training_type_plugin.lightning_module_state_dict()
 
     def barrier(self, name: Optional[str] = None) -> None:
+        """
+        .. deprecated:: v1.5
+            This method is deprecated in v1.5 and will be removed in v1.6.
```

Comment:
Is this an error or was there a discussion to only deprecate for 1 minor version?

Reply:
@carmocca here's the discussion thread: https://github.com/PyTorchLightning/pytorch-lightning/pull/9677#discussion_r715848175